public/data/2018-10-01/employees.csv

```diff
@@ -9213,6 +9212,7 @@ ethnicityId,salaryBucketId,departmentId,gender
 1,1,22,F
 1,1,22,F
 1,2,44,F
+3,1,22,F
```

Comment:
Most of these just look like the order changed. I will investigate.

Reply:
After I changed the data-fetch script, and used it to pull in all of the files we have on the open data portal, I re-ran the data-import script, and it creates all of these files.  So, the changes might just be a change in order, but some of the data might be a bit different as well - just depending on what we have in the open data portal, versus what the website was originally using.